pkgs/top-level/all-packages.nix

```diff
@@ -12502,7 +12502,9 @@ with pkgs;
   sigil = libsForQt5.callPackage ../applications/editors/sigil { };
 
   signalbackup-tools = darwin.apple_sdk_11_0.callPackage
-    ../applications/networking/instant-messengers/signalbackup-tools { };
+    ../applications/networking/instant-messengers/signalbackup-tools {
+      inherit (darwin.apple_sdk.frameworks) Security;
```

Comment:
Please use `darwin.apple_sdk.frameworks.Security` directly, see https://github.com/NixOS/nixpkgs/issues/204303

Reply:
Thanks, I wasn't aware of that. Fixed.